treeAnalysis/jetresolutionhistos.cxx

```diff
@@ -73,6 +81,8 @@ TH2F *h2D_truth_eta_phi[njettypes] = {NULL};
 TH2F *h2D_cluster_eta_phi[njettypes] = {NULL};
 
 
+double momentum(fastjet::PseudoJet jet) {return TMath::Sqrt(jet.px() * jet.px() + jet.py() * jet.py() + jet.pz() * jet.pz());}
```

Comment:
Just fyi - better to use `std::sqrt` . Many TMath functions are basically around for historical reasons

Reply:
Good to know, thanks.  If you see stuff like this in the future feel free to hold off on merging it, I'd like the use best practices where I can.

Reply:
Sure, will do - I've been merging quickly for the sake of moving quickly, but I can move a bit more deliberately

Reply:
I was a little distracted earlier - the better answer is that you don't need this function - just use `PseudoJet::modp()`

Reply:
Ah perfect, somehow when I looked over the fastjet documentation I missed that!